packages/react-native-executorch/common/rnexecutorch/models/speech_to_text/asr/ASR.cpp

```diff
@@ -246,19 +244,12 @@ std::vector<Segment> ASR::transcribe(std::span<const float> waveform,
   return results;
 }
 
-std::vector<float> ASR::encode(std::span<const float> waveform) const {
-  constexpr int32_t fftWindowSize = 512;
-  constexpr int32_t stftHopLength = 160;
-  constexpr int32_t innerDim = 256;
-
-  std::vector<float> preprocessedData =
-      dsp::stftFromWaveform(waveform, fftWindowSize, stftHopLength);
```

Comment:
If `stftFromWaveform` will be removed, this function should be removed completely from codebase since it is not used anywhere else.